### DISCUSSION TITLE
changing non-linear tolerance during simulation

### DISCUSSION THREAD

**Discussion by sinaomrani96** [idaholab/moose#24592] [Q&A General] [answered]
Hi,
Hope you are doing well.

I wanted to know that if we can change the non-linear tolerance during the simulation. In my system, I add an extra phase to simulation which add to non-linearity of system and I was wondering if this is possible. The alternative way would be restart/recover method with a more loose tolerance, I guess. Thanks.

**Comment by GiudGiud** [comment, marked as answer]
Hello

No we currently cant change that number. It's set in the input file and we can't modify it

Guillaume
